x/checkpoint/client/cli/tx.go

```diff
@@ -72,41 +72,26 @@ func CheckpointTxCmd() *cobra.Command {
 			}
 
 			if isAutoConfigure {
-				var checkpointProposer hmTypes.Validator
-				proposerBytes, _, err := clientCtx.Query(fmt.Sprintf("custom/%s/%s", types.StakingQuerierRoute, types.QueryCurrentProposer))
+				stakingQueryClient := stakingtypes.NewQueryClient(clientCtx)
+				validatorSet, err := stakingQueryClient.ValidatorSet(context.Background(), &stakingtypes.QueryValidatorSetRequest{})
```

Comment:
Let's add get current proposer methods in StakingQueryClient instead of doing this way. Wdyt?